pkg/vpcmodel/explainabilityConnectivity.go

```diff
@@ -29,9 +29,30 @@ type rulesConnection struct {
 	egressRules  rulesInLayers
 }
 
-// loadBalancerDennyEgressRule - is a rule applied to all private IPs of a given load balancer:
+// the load balancer is a rule applied to all private IPs of a given load balancer:
 // these private IPs can only connect to pool members of the load balancer
-type loadBalancerDenyEgressRule bool
+type LoadBalancerRule struct {
```

Comment:
Is this specific to IBM?
If it is, wouldn't it make sense to implement in the VPC package? and keep here only some abstraction?

Reply:
Load balancer is not an IBM concept.
currently, we have LoadBalancer as an interface at vpcmodel
and LoadBalancer as a struct in the ibmvpc.

I guess that LoadBalancerRule should be an interface too and implemented differently on each vendor,  but I do not have all the information.
does load balancer from other vendors effect on the connectivity, I assume yes, but how?
there is also high chance that all LoadBalancerRules from all vendors will share the same behavior, and there will be no need for interface.

for now, I wrapped  all in functions, to make it easy to change it to an interface later